palantir-java-format/src/test/java/com/palantir/javaformat/NewlinesTest.java

```diff
@@ -68,7 +69,7 @@ public void terminalOffset() {
         it.next();
         try {
             it.next();
-            Assert.fail();
+            fail("fail");
         } catch (NoSuchElementException e) {
             // expected
         }
```

Comment:
@carterkozak we might want to have specific logic that deals with this!

Reply:
@dansanduleac are you hoping for a assertThatThrownBy here? I thought this was fine? (it's only meant to be a representative example anyway right?)

Reply:
I missed the notification on this, sorry.
Agree a refactor to assertThatThrownBy is ideal, but it would take a bit of time to implement